src/CONST.ts

```diff
@@ -600,6 +600,7 @@ const CONST = {
             ADMINS: '#admins',
         },
         STATE: {
+            OPENED: 'OPENED',
```

Comment:
These state names are not really useful, I think we can remove them later and only keep the state and status numbers, but that would be other PR

Reply:
I believe we typically call this state `OPEN` right? Not `OPENED`.